bliss/conf/base_config.yaml

```diff
@@ -170,19 +171,38 @@ surveys:
             - run: 94
               camcol: 1
               fields: [12]
+        prior_config: ${prior}
         psf_config:
             pixel_scale: 0.396
             psf_slen: 25
-        n_bands: 5
     decals:
         _target_: bliss.surveys.decals.DarkEnergyCameraLegacySurvey
-        decals_dir: ${paths.decals}
+        dir_path: ${paths.decals}
         sky_coords: # in degrees
             # brick '3366m010' corresponds to SDSS RCF 94-1-12
             - ra: 336.6643042496718
               dec: -0.9316385797930247
         bands: ${encoder.bands}
         reference_band: 1 # DECaLS r-band, DECaLS-indexed (via header)
+        prior_config: ${prior}
+    des:
+        _target_: bliss.surveys.des.DarkEnergySurvey
+        dir_path: ${paths.des}
+        image_ids:
+            - sky_coord:
+                ra: 336.6643042496718
+                dec: -0.9316385797930247
+              decals_brickname: "3366m010"
+              ccdname: "S28"
+              g: "decam/CP/V4.8.2a/CP20171108/c4d_171109_002003_ooi_g_ls9"
+              r: "decam/CP/V4.8.2a/CP20170926/c4d_170927_025457_ooi_r_ls9"
+              i: ""
+              z: "decam/CP/V4.8.2a/CP20170926/c4d_170927_025655_ooi_z_ls9"
+        reference_band: 1 # DES r-band, DES-indexed
+        prior_config: ${prior}
```

Comment:
Conceptually, the prior is a property of the simulator (i.e., the generative model), not the survey. (Multiple priors could be appropriate for the same survey depending on the type of analysis.)

Reply:
Also, better to call the field `prior` not `prior_config` (everything in a yaml file is a conf)

Reply:
`survey` could be a property of `simulator` because the data we're simulating is semi-synthetic, i.e., based on real data.

Reply:
Done.